Automattic/wp-calypso
# Renaming Simple Payments to Pay with PayPal Button

We are looking to rename the Simple Payments block to Pay with PayPal button - p2y3YZ-43P

This change shouldn't affect the simple payments block behaviour. The changes here are only intended to change the use of public facing `Simple Payments` terminology to `Pay with PayPal button`.

Page | Before | After
--- | --- | ---
`/plans/:site` | <img width="987" src="https://user-images.githubusercontent.com/1233880/85399040-6ab0ad80-b556-11ea-8856-c2d4d287d984.png"> | <img width="987" alt="Screen Shot 2020-06-23 at 13 35 48" src="https://user-images.githubusercontent.com/1233880/85399098-8ae06c80-b556-11ea-83b2-95018555aef8.png">
`/post/:site` |  <img width="214" alt="Screen Shot 2020-06-23 at 13 43 48" src="https://user-images.githubusercontent.com/1233880/85399790-aa2bc980-b557-11ea-93e2-c5bf779fd970.png"> | <img width="211" alt="Screen Shot 2020-06-23 at 13 46 06" src="https://user-images.githubusercontent.com/1233880/85399937-e9f2b100-b557-11ea-95bb-238115c6cb83.png">
`/post/:site` | <img width="769" alt="Screen Shot 2020-06-23 at 13 43 57" src="https://user-images.githubusercontent.com/1233880/85399806-b4e65e80-b557-11ea-9cdf-cf2dda35d859.png"> | <img width="713" alt="Screen Shot 2020-06-23 at 13 44 21" src="https://user-images.githubusercontent.com/1233880/85399814-b9ab1280-b557-11ea-9dc3-2a98dd9523e0.png">
`/earn/:site` | <img width="540" alt="Screen Shot 2020-06-23 at 16 02 53" src="https://user-images.githubusercontent.com/1233880/85413387-0e0bbd80-b56b-11ea-9c85-ad00b13fed59.png"> | <img width="531" alt="Screen Shot 2020-06-23 at 16 05 45" src="https://user-images.githubusercontent.com/1233880/85413701-6cd13700-b56b-11ea-941d-e6fb3c298c43.png">
`/plans/my-plan/:site` | <img width="555" alt="Screen Shot 2020-06-23 at 16 13 34" src="https://user-images.githubusercontent.com/1233880/85414545-88890d00-b56c-11ea-80a3-e29c7b15c834.png"> | <img width="534" alt="Screen Shot 2020-06-23 at 16 18 25" src="https://user-images.githubusercontent.com/1233880/85415081-2ed51280-b56d-11ea-9db9-f75cfa8bd575.png">

#### Testing instructions

Go through the list of screenshots above and make sure the new copy is used only when the `earn/rename-payment-blocks` flag is enabled.
